card/mod/05_standard/lib/file_uploader.rb

```diff
@@ -146,7 +146,7 @@ def retrieve_dir
   end
 
   def mod_file?
-    @mod ||= model.mod_file?
+    model.mod_file?
```

Comment:
this is kind of an expensive lookup, no?  Is it safe to assume this is getting incorrectly cached at some point?  Is it possible we just need to clear it?

The builds appear to be getting slower, but it may not be because of this PR.